### DISCUSSION TITLE
How do I interpret the Pull Shark badge?

### DISCUSSION THREAD

**Discussion by beingshahidali** [community/community#140747] [Pull Requests] [answered]
<img src="https://github.com/user-attachments/assets/25db76ee-bc9a-4748-a128-036702284197" alt="Pull Shark Badge" style="width: 150px; height: auto;">

<div>
  <h2>How do I interpret the Pull Shark badge?</h2>
  
  <div>
    <p>I just unlocked the Pull Shark badge, and while I love GitHub's gamification of PRs, I'm a bit confused by how it works. 🦈</p>
  </div>

  <ul>
    <li>I’ve earned the <strong>Pull Shark x3</strong>, but I’m unsure what the "x3" represents. Does that mean I've had 3 PRs merged? Or does it count more?</li>
  </ul>

  <div>
    <p>From what I understand:</p>
    <ul>
      <li><strong>Bronze</strong> unlocks after 2 PRs are merged.</li>
      <li><strong>Silver</strong> comes after 16 PRs are merged.</li>
      <li><strong>Gold</strong> comes after 128 PRs are merged.</li>
    </ul>
  </div>

  <div>
    <p>For anyone aiming to get more Pull Shark badges or who want tips on leveling up your PR game, follow me for updates! 🔥</p>
  </div>
</div>

**Comment by ben-lotze** [comment, marked as answer]
It means that you unlocked this badge 3 times. 
- 1st time after the 2nd PR ("First unlocked")
- 2nd time after the 16th PR ("Bronze" unlocked)
- 3rd time after the 128th PR ("Silver" unlocked)

You can get it a 4th time after 1024 PRs (which will unlock "Gold"), see my profile:
<img width="472" alt="Pull Shark achievement" src="https://github.com/user-attachments/assets/d8327021-bc2f-4220-9f26-99f1a17ab936">

**Reply by fdwr** [reply]
Egads, 1024!? At 175 (roughly 35 per year), that's only 24 more years for me to get gold 🏆... 😅
